webpack.config.renderer.dev.js

```diff
@@ -50,6 +50,23 @@ export default merge.smart(baseConfig, {
   module: {
     rules: [
       {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            cacheDirectory: true,
+            plugins: [
```

Comment:
Got it. So we should move `es2015-classes` to the renderer webpack config and remove `class-properties`?